docker-compose-deploy.yml

```diff
@@ -0,0 +1,12 @@
+# Compose file for local dev, not for prod
```

Comment:
File is named deploy but comment says it's not for prod

Reply:
true